clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -753,6 +753,10 @@ public void subscribe(Pattern pattern, ConsumerRebalanceListener listener) {
     public void subscribe(Pattern pattern) {
         delegate.subscribe(pattern);
     }
+    @Override
+    public void subscribe(SubscriptionPattern pattern, ConsumerRebalanceListener callback) {}
+    @Override
+    public void subscribe(SubscriptionPattern pattern) {}
```

Comment:
This method needs javadocs describing what this method does. Please also add an empty line before the method.